src/lib/fork_id/fork_id.mli

```diff
@@ -0,0 +1,18 @@
+(* fork_id.mli *)
+
+[%%versioned:
+module Stable : sig
+  module V1 : sig
+    type t [@@deriving sexp]
+  end
+end]
+
+type t = Stable.Latest.t [@@deriving sexp]
+
+val required_length : int
+
+val create : string -> t
```

Comment:
Should be named `create_exn` since it throws exceptions.

We should also have a `string -> t option` variant of this, so that we don't crash when we read fork ids from objects received over the network.

Reply:
`create_opt` added